README.md

```diff
@@ -12,13 +12,24 @@ The [Tech Radar](https://techradar.red-gate.com) describes the technology landsc
 * Platforms. Things that we build software on top of such as .NET, SQL Server, Windows (etc).
 * Techniques. These include elements of a software development process, such as experience design; and structuring software, such as microservices.
 
-Items on the Tech Radar should only be captured if the cost of change is high or the benefits of standardizing outweigh the drawbacks. Each item on the tech radar should be there for the the good of Redgate or our customers.
+Items on the Tech Radar should only be captured if the cost of change is high or the benefits of standardizing outweigh the drawbacks. Each item on the tech radar should be there for the the good of Redgate or our customers. 
+
+We are not producing a tech radar for the general tech community to consume, the tech radar is to guide our teams to build better software, so favour things that are going to be actioned over theoretical "this could be useful".
 
 Each technology choice is positioned in one of four areas.
-* Adopt – We feel strongly that Redgate should be adopting these items. It’s the default choice.
-* Trial – Worth pursuing. It’s important to understand how to build up this capability. Redgate teams can try this technology on a project that can support the risk.
-* Assess – Worth exploring with the goal of understanding how it will affect Redgate
-* Hold - Proceed with caution
+* **Adopt** – The technology/technique is recommended for use by the majority of teams. Consider it the default choice for new work.
+  * What resources have we built to help guide people to adopting this?
+  * Guidance on migration from existing technology.
```

Comment:
Minor: initially I was confused reading these two sub bullet points. Perhaps it'd be clearer to say something like: "To move something into 'Adopt': "People have the resources they need to adopt this" and "Where applicable, guidance is available on how to migrate from existing technology"? E.g., say what is required instead of asking an open question.

Reply:
I like your suggestion for this - I was tripped up by this part too initially.